routemodel/data_retrieval/get_weather.py

```diff
@@ -0,0 +1,46 @@
+def get_weather():
+    
+    import sys
+    import os.path
+    sys.path.append(os.path.dirname(__file__))
+    import requests
+    import csv
+    import json
+
+    from config import WEATHER_API_KEY
+
+    one_call_base = 'https://api.openweathermap.org/data/2.5/onecall?'
+    path = os.path.join(os.path.dirname(__file__), '..', 'routes\ASC2021\ASC2021_draft.csv')
```

Comment:
I would encourage you to add this as a parameter to the function so it can be passed/easily changed